local-cli/runWpf/runWpf.js

```diff
@@ -1,45 +1,33 @@
 'use strict';
 
 const chalk = require('chalk');
-const build = require('./utils/build');
-const deploy = require('./utils/deploy');
+const automatedVS = require('../automated-vs/index');
+const WPFSolution = automatedVS.WPFSolution;
+const buildAndDeployUtils = require('../utils/buildAndDeployUtils');
+
+const minWinSDK = '8.1.0.0';
 
 function runWpf(config, args, options) {
-  // Fix up options
+
   options.root = options.root || process.cwd();
-  if (options.debug && options.release) {
-    console.log(chalk.red('Only one of "debug"/"release" options should be specified'));
-    return;
-  }
 
-  const slnFile = build.getSolutionFile(options);
+  const slnFile = buildAndDeployUtils.getSolutionFile(options, 'wpf');
   if (!slnFile) {
-    console.error(chalk.red('Visual Studio Solution file not found. Maybe run "react-native wpf" first?'));
-    return;
-  }
-
-  try {
-    build.restoreNuGetPackages(options, slnFile);
-  } catch (e) {
-    console.error(chalk.red('Failed to restore the NuGet packages'));
+    console.error(chalk.red('Visual Studio Solution file not found. Make sure your root directory contains the wpf folder.'));
```

Comment:
previous error message `'Visual Studio Solution file not found. Maybe run "react-native wpf" first?'` didn't actually describe the problem. `react-native run-wpf` wouldn't work if the user hadn't run `react-native wpf` first.